src/studio/nuxeo-studio/tutorials/how-to-work-with-git-and-studio-project.md

```diff
@@ -26,19 +26,19 @@ Watch the [WebUI Customization and Nuxeo Frontend Development](https://universit
 1. A **running instance** of Nuxeo server on your machine (http://localhost:8080/nuxeo/) to deploy your configuration
 2. The **nuxeo CLI** development tool to execute the replication operation. Please refer to [Nuxeo CLI]({{page space='nxdoc' page='nuxeo-cli'}}) to install it on your computer
 3. A **Studio project** to create the project which will be cloned locally
-4. The **Branch Management** feature activated on your Studio Project. If you don't have it, you can only pull your coding without pulling it
+4. The **Branch Management** feature activated on your Studio Project. If you don't have it, you can only pull your Studio project without pushing the changes you've made locally. In this case, you have to manally copy your contribution in the Nuxeo Studio Designer interface, in the Resources tab.
```

Comment:
Generally, we have to use the full form whenever we do negation like **do not** or  with an auxiliary **you have** this is more formal.

-> manually